helm/stunner-prometheus/templates/prometheus-rule.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    prometheus: prometheus
+    role: alert-rules
+  name: prometheus-rules
+  namespace: {{ .Values.namespace }}
+spec:
+  groups:
+  #few alert rules example
+  - name: ./example.rules
+    rules:
+    #this one fires immediately
+    - alert: ExampleAlert
+      expr: vector(1)
+  - name: ./ctrlr_list.rules
+    rules:
+    #if more then five requests came trough, fires
+    - alert: listener-over-five-requests
```

Comment:
Remove this part, please. If I'm correct it was copied from the l7mp-prometheus chart. Also, I don't know if we even need a PrometheusRule or not.

Reply:
Thanks! Fixed in [74d9e1d](https://github.com/l7mp/stunner-helm/pull/5/commits/74d9e1ddd72ff1f59c52c9f49d25c7f1b28932c5).